manager/container.go

```diff
@@ -40,6 +41,10 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	JitterFactor = 1.0
```

Comment:
This is a bit redundant isn't it? Remove for now, add back in later if different from constant 1?